src/code/z_play.c

```diff
@@ -180,6 +180,10 @@ void Play_Destroy(GameState* thisx) {
     this->state.gfxCtx->callback = NULL;
     this->state.gfxCtx->callbackParam = NULL;
 
+#ifdef ENABLE_MOTION_BLUR
+    Play_DestroyMotionBlur();
+#endif
+
     SREG(91) = 0;
```

Comment:
Be careful with `#define R_MOTION_BLUR_ENABLED                SREG(91)` since OoT has `SREG(91)` set in two places. Seems unused otherwise, but there's also a `SREG(91) = -1` to take care of. Other `SREG(9X)` seem fine

Reply:
changed it to ``SREG(95)``